test-suite/output/RealSyntax.out

```diff
@@ -20,3 +20,18 @@ eq_refl : -1e-4 = -1e-4
      : -1e-4 = -1e-4
 eq_refl : -0.50 = -0.50
      : -0.50 = -0.50
+eq_refl : -26 = -26
+     : -26 = -26
+eq_refl : 2860 / IZR (BinIntDef.Z.pow_pos 2 8) = 2860 / IZR (Z.pow_pos 2 8)
```

Comment:
Seems a little bit strange to me that printing displays this weird format; is this intended?

Reply:
Indeed, the hexadecimal printer for R is not implemented. Maybe not worth investing too much into that OCaml code if it gets replaced by a Coq version in #12218